handler.go

```diff
@@ -85,6 +85,7 @@ func newHandler(handlerSymbol interface{}) lambdaHandler {
 			eventType := handlerType.In(handlerType.NumIn() - 1)
 			event := reflect.New(eventType)
 
+			// TODO: I don't think this is even possible
```

Comment:
What does this refer to?

Reply:
I don't think it's possible for `json.Unmarshal` to error. The thing being unmarshalled has just come from `json.Marshal`. This is because... It is necessary to wrap the function in order to accomodate a variety of function signatures, this results in all payloads being received as `interface{}` by the wrapping function. We then need to marshal that payload, and unmarshal it again to populate the concrete type.

Reply:
Got it, thank you!